ncm-metaconfig/src/main/metaconfig/slurm/pan/acct_gather.pan

```diff
@@ -0,0 +1,19 @@
+unique template metaconfig/slurm/acct_gather;
+
+include 'components/metaconfig/config';
+include 'metaconfig/slurm/schema';
+
+bind "/software/components/metaconfig/services/{/etc/slurm/acct_gather.conf}/contents" = slurm_acct_gather_conf;
+
+variable SLURM_DAEMONS = {
```

Comment:
this variable egenration should be moved to a separate template and included where needed